src/stores/AppStore.tsx

```diff
@@ -30,11 +30,11 @@ export const AppStore = types
     showSubstances: types.boolean,
     // which views we allow in the organism boxes
     // Default: ['None', 'Organism', 'Cell', 'Receptor'], set with `?availableViews=Organism,Cell`
-    availableViews: types.array(types.string)
+    _availableViews: types.array(types.string)
```

Comment:
@sfentress I have this problem in other parts of the code too but I don't really like revealing `availableViews` and also `getAvailableViews()` when really only the latter should be used. How do you feel about this approach? I know `_private` isn't a great convention but I can't find another way.